fix: LEAP-2051: Don't allow to resize spans in readonly mode

hlomzik opened the pull request:
Predictions and View All are readonly, we should not allow to resize spans there. Simple check added to check for readonly. It's reactive so triggered every time you go to View All.

(+ a little import sorting)

hlomzik committed:
fix: LEAP-2051: Don't allow to resize spans in readonly mode

Predictions and View All are readonly, we should not allow to resize spans there.
Simple check added to check for readonly. It's reactive so triggered every time you
go to View All.

hlomzik committed:
Sort imports with Biome

MihajloHoma commented:
/git merge 
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/14662247639)
> Successfully merged:  2 files changed, 53 insertions(+), 25 deletions(-)

MihajloHoma committed:
Merge branch 'develop' into 'fb-leap-2051/readonly-spans'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14662247639

MihajloHoma commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/14662294931)

MihajloHoma committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14662294931

hlomzik commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/14665339563)
> Successfully merged:  4 files changed, 45 insertions(+), 29 deletions(-)

hlomzik committed:
Merge branch 'develop' into 'fb-leap-2051/readonly-spans'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/14665339563

hlomzik committed:
Temporary test fix
